src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
-
 // write your code here
+
+// Отримуємо таблицю
+
+const table = document.querySelector('table');
+
+// Отримуємо всі рядки в таблиці (включаючи шапку)
+
+const rows = table.querySelectorAll('tr');
+
+rows.forEach((row) => {
+  // Отримуємо другу колонку (індекс 1)
+  const secondColumm = row.children[1].cloneNode(true);
```

Comment:
There is a typo in the variable name `secondColumm`. It should be `secondColumn` to correctly reflect the intended meaning.